src/Microsoft.Data.SqlClient/tests/ManualTests/Microsoft.Data.SqlClient.ManualTesting.Tests.csproj

```diff
@@ -233,6 +233,10 @@
     <Compile Include="TracingTests\DiagnosticTest.cs" />
     <Compile Include="TracingTests\FakeDiagnosticListenerObserver.cs" />
   </ItemGroup>
+  <!--.Net core 3.0 & .Net standard 2.1 and above-->
+  <ItemGroup Condition="'$(TargetGroup)'=='netcoreapp' AND !$(TargetFramework.StartsWith('netcoreapp2')) AND !$(ReferenceType.Contains('netcoreapp2')) AND !$(ReferenceType.Contains('NetStandard2.0'))">
```

Comment:
The ReferenceType conditions are not needed. When running the tests against .NET Standard 2.1, the target framework is set to `netcoreapp3.1`.